From 9c181d52499152c9a09d5fcced281a4940862494 Mon Sep 17 00:00:00 2001
From: liduanjun <liduanjun@uniontech.com>
Date: Wed, 23 Jun 2021 10:27:36 +0800
Subject: [PATCH] fix: add 709 graphic card support
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
select csmicro other than modesetting for 709 graphic card
Log: 使709显卡默认使用 csmicro_drv.so 而不是 modesetting_drv.so
Signed-off-by: liduanjun <liduanjun@uniontech.com>
Change-Id: I25ec75c5b9b4e5313c3e2197e1f932beb3d94433

--- xorg-server-1.20.4.11.orig/hw/xfree86/common/xf86pciBus.c
+++ xorg-server-1.20.4.11/hw/xfree86/common/xf86pciBus.c
@@ -1303,6 +1303,9 @@ xf86VideoPtrToDriverList(struct pci_devi
         else
             driverList[0] = "xgi";
         break;
+    case 0x0709:
+        driverList[0] = "csmicro";
+        break;
     case 0x1d17:
         switch (dev->device_id) {
         case 0x3a03:
